test/test.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const {TestRunner, Reporter, Matchers} = require('@pptr/testrunner');
```

Comment:
curious how this suits non-pptr use cases compared to jest/mocha :)

Reply:
@patrickhulce We'll see :-) I like it because it has zero NPM dependencies.

It also has zero magic: no global object injections, no special executable that does something with your test file. Tests are just a node.js script - run with `node test/test.js`, debug with `node --inspect-brk test/test.js`.